src/etos_lib/logging/filter.py

```diff
@@ -28,7 +28,9 @@ def get_current_otel_trace_id() -> str:
     If OpenTelemetry is not enabled, this function will return "0".
     """
     current_span = trace.get_current_span()
-    return str(current_span.get_span_context().trace_id)
+    trace_id = current_span.get_span_context().trace_id
+    trace_id = hex(trace_id).replace("0x", "")
```

Comment:
There is a convenince method for this in `opentelemetry.trace` called `format_trace_id()`
https://opentelemetry-python.readthedocs.io/en/latest/api/trace.html#opentelemetry.trace.format_trace_id